testdata/scripts/make_vds.py

```diff
@@ -0,0 +1,284 @@
+        config.cdp.xline_spacing,
+    )
+    metaData.setMetadataString(
+        openvds.KnownMetadata.surveyCoordinateSystemUnit().category,
+        openvds.KnownMetadata.surveyCoordinateSystemUnit().name,
+        "m",
+    )
+    metaData.setMetadataString(
+        openvds.KnownMetadata.surveyCoordinateSystemCRSWkt().category,
+        openvds.KnownMetadata.surveyCoordinateSystemCRSWkt().name,
+        "utmXX",
+    )
+    metaData.setMetadataString(
+        openvds.KnownMetadata.importInformationInputFileName().category,
+        openvds.KnownMetadata.importInformationInputFileName().name,
+        config.import_info.file_name,
+    )
+    metaData.setMetadataString(
+        openvds.KnownMetadata.importInformationDisplayName().category,
+        openvds.KnownMetadata.importInformationDisplayName().name,
+        config.import_info.display_name,
+    )
+    metaData.setMetadataString(
+        openvds.KnownMetadata.importInformationInputTimeStamp().category,
+        openvds.KnownMetadata.importInformationInputTimeStamp().name,
+        config.import_info.input_timestamp,
+    )
+    metaData.setMetadataString(
+        openvds.KnownMetadata.importInformationImportTimeStamp().category,
+        openvds.KnownMetadata.importInformationImportTimeStamp().name,
+        config.import_info.import_timestamp,
+    )
+    metaData.setMetadataString(
+        openvds.KnownMetadata.importInformationInputFileSize().category,
+        openvds.KnownMetadata.importInformationInputFileSize().name,
+        config.import_info.input_file_size,
+    )
+    vds = openvds.create(
+        vds_filename,
+        "",  # connection string
+        layoutDescriptor,
+        axisDescriptors,
+        channelDescriptors,
+        metaData,
+        compressionMethod,
+        compressionTolerance,
+    )
+
+    manager = openvds.getAccessManager(vds)
+    accessor = manager.createVolumeDataPageAccessor(
+        openvds.DimensionsND.Dimensions_012,
+        0,  # LOD
+        0,  # channel
+        8,  # Max pages in cache
+        openvds.IVolumeDataAccessManager.AccessMode.AccessMode_Create,
+    )
+
+    for c in range(accessor.getChunkCount()):
+        page = accessor.createPage(c)
+        buf = np.array(page.getWritableBuffer(), copy=False)
+        (min, max) = page.getMinMax()
+        buf[:, :, :] = config.data[min[2] : max[2], min[1] : max[1], min[0] : max[0]]
+
+        page.release()
+
+    accessor.commit()
+    openvds.close(vds)
+
+
+def make_well_known_custom_axis_order(segy_filename: str, vds_filename: str) -> None:
+    src = segyio.open(segy_filename)
+    data = np.transpose(segyio.tools.cube(src), (2, 0, 1))
+    axes = [Config.Axis.xline(src), Config.Axis.inline(src), Config.Axis.time(src)]
+    cdp = Config.CDP.from_segy(src)
+    import_info = Config.ImportInfo.from_file(segy_filename)
+    config = Config(data, axes, cdp, import_info)
+    create_vds(vds_filename, config)
+
+
+def make_unknown_axis(vds_filename: str) -> None:
```

Comment:
Maybe split those two into their own files named the same as final .vds files (into current directory, for simplicity of import) and add simple `__main__` functions?

It would have been nicer to split those to corresponding directories, but then `from make_vds import *` won't be enough for import, so the simple solution is better.